includes/Modules/Ads.php

```diff
@@ -85,7 +93,64 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script_Data(
+				'googlesitekit-ads-pax-config',
```

Comment:
The new scripts + data should only be added when the feature is enabled (see first line of AC).